docs/source/host-collect-analyze/run.md

```diff
@@ -33,6 +33,22 @@ spec:
         collectorName: "run-with-shell"
         command: "sh"
         args: ["-c", "du -sh | sort -rh | head -5"]
+    # Multiline shell script
+    - run:
+        collectorName: "hostnames"
+        command: "sh"
+        args:
+          - -c
+          - |
+            echo "\$HOSTNAME = $HOSTNAME"
```

Comment:
This one does not work

~$ cat support-bundle-2023-02-17T14_46_11/host-collectors/run-host/hostnames.txt
$HOSTNAME = 
hostname = camila-ubuntu-2004-similar-but-contairnd
/proc/sys/kernel/hostname = camila-ubuntu-2004-similar-but-contairnd
uname -n = camila-ubuntu-2004-similar-but-contairnd

Reply:
Yer, it won't. The new shell created won't have access to the parent shell's environment. Also, `HOSTNAME` is a bash specific variable. I'll update the doc